jekyll/_cci2/schedule-pipelines-with-multiple-workflows.adoc

```diff
@@ -17,22 +17,34 @@ contentTags:
 [#introduction]
 == Introduction
 
-As a scheduled pipeline is a triggered pipeline, and it will run every workflow in the configuration. You can set conditionals to keep the scheduled pipelines running only when you want them to run. One way is through built-in pipeline values, and one way is through custom pipeline parameters.
+As a scheduled pipeline is a triggered pipeline, and it will run every workflow in the configuration. You can filter workflows with set conditionals to keep the scheduled pipelines running only when you want them to run. One way is through built-in pipeline values, and one way is through custom pipeline parameters.
```

Comment:
I'm finding this paragraph difficult to read. Maybe can rephrase... something like "Scheduled pipelines are simply triggered pipelines, and by default, for each trigger, every workflow that has been configured will run. You can filter workflows with conditionals to control which workflows will run when a scheduled pipeline is triggered. One way is through built-in pipeline values, and one way is through custom pipeline parameters"